keel-sql/src/test/kotlin/com/netflix/spinnaker/keel/sql/spring/SpringStartupTests.kt

```diff
@@ -0,0 +1,44 @@
+package com.netflix.spinnaker.keel.sql.spring
+
+import com.netflix.spinnaker.keel.KeelApplication
+import com.netflix.spinnaker.keel.persistence.ResourceRepository
+import com.netflix.spinnaker.keel.sql.SqlLock
+import com.netflix.spinnaker.keel.sql.SqlResourceRepository
+import com.netflix.spinnaker.keel.sync.Lock
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment.MOCK
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import strikt.api.expectThat
+import strikt.assertions.isA
+
+@ExtendWith(SpringExtension::class)
+@SpringBootTest(
+  classes = [KeelApplication::class],
+  webEnvironment = MOCK,
+  properties = [
+    "sql.enabled=true",
+    "sql.connection-pools.default.jdbc-url=jdbc:h2:mem:keel;MODE=MYSQL",
+    "sql.migration.jdbc-url=jdbc:h2:mem:keel;MODE=MYSQL"
+  ]
+)
+internal class SpringStartupTests {
```

Comment:
Thought it's somewhat valuable to have this test. The `keel-redis` module has an equivalent.

Reply:
nice, yes big +1